docs/src/pages/components/card/react-auto-prop-table-card.mdx

```diff
@@ -0,0 +1,121 @@
+import { Expander, ExpanderItem } from '@aws-amplify/ui-react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+## Props
+
+<Expander type="multiple" defaultValue={['Props']}>
+  
+<ExpanderItem title="Amplify UI Props" value="Props">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
+| ---- | ---- | ------- | ----------- |
+| ariaLabel | string | / |  |
+| ariaValuetext | string | / |  |
+| as | ElementType | / |  |
+| className | string | / | Additional CSS class name for component |
+| id | string | / | Unique identifier |
+| isDisabled | boolean | / |  |
+| ref | Ref&#60;HTMLDivElement&#62; | / |  |
+| role | React.AriaRole | / |  |
+| style | CSSProperties | / |  |
+| testId | string | / | Used to provide a &#96;data-testid&#96; attribute for testing purposes |
+| variation | CardVariations | / |  |
+
+      `}
+    />
+
+  </ExpanderItem>
+
+<ExpanderItem title="Layout" value="Layout">
+    <ReactMarkdown
+      remarkPlugins={[remarkGfm]}
+      children={`
+| Name | Type | Default | Description |
```

Comment:
Same comment on this, can't we save this into a shared file and import it instead? I think each of these files should just have the unique props in them.

Reply:
The Styles props has only 1 combination, but the Layout props have at least 2 combinations...so we may not want to use a shared file since it's hard to say what a new primitive will use

Reply:
I think it's not necessary to use a `shared` file here, because the shared file is to make it easier when manually creates a file. In our case, it's auto-generated, there's no benefit of using a shared file. They will be build into the target file at the build time, right?